<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QPushButton" name="becomesmaller">
      <property name="text">
       <string>变小</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(245, 245, 245);  /* 设置背景色为灰色 */
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="todayButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>今日任务</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="expectsButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>预定任务</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QToolButton" name="extendButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>已延期</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QToolButton" name="yestButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>昨日任务</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QToolButton" name="longtermButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>长期任务</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>294</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QToolButton" name="ddlButton">
         <property name="styleSheet">
          <string notr="true">QToolButton{   
	border-top: 3px outset transparent;         /* 上边框设置*/
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 150px;
    min-height: 40px;
	background-color: rgb(245, 245, 245);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(235, 235, 235);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 5px outset rgb(23, 117, 217);  /* 左边框宽度3像素、突出显示 */
	background-color: rgb(235, 235, 235);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
         </property>
         <property name="text">
          <string>截止日期分布</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
